release-notes/opensearch-dashboards-search-relevance.release-notes-2.10.0.0.md

```diff
@@ -0,0 +1,12 @@
+## Version 2.10.0.0 Release Notes
+
+Compatible with OpenSearch 2.10.0
+
+### Enhancements
+* Changing select index error message to be more informative ([#270](https://github.com/opensearch-project/dashboards-search-relevance/pull/270)) ([#275](https://github.com/opensearch-project/dashboards-search-relevance/pull/275))
+
+### Infrastructure
+* Changing anomaly-detection Dependency Back to search-processor ([#258](https://github.com/opensearch-project/dashboards-search-relevance/pull/258))([#259](https://github.com/opensearch-project/dashboards-search-relevance/pull/259))
+* Adding a developer guide ([#268](https://github.com/opensearch-project/dashboards-search-relevance/pull/268)) ([#269](https://github.com/opensearch-project/dashboards-search-relevance/pull/269))
```

Comment:
should this go to Documentation? https://github.com/opensearch-project/opensearch-plugins/blob/main/RELEASE_NOTES.md#documentation

Reply:
Sure!